docs/deploy.rst

```diff
@@ -55,8 +55,7 @@ exodus-pipeline.yaml, run by the `AWS CLI`_ command
         Available values: dev, stage, prod
         Default: dev
     - oai:
-        The origin access identity ID associated with the environment,
-        created alongside the environment's bucket-table pair.
+        An `AWS CloudFront`_ origin access identity.
```

Comment:
This value here should be the same one used for the storage step. Maybe adding another section for oai creation and than populating the value when running the other two steps. WDYT?

Reply:
Sorry, I don't think I follow. Are you referring to the sentence being repeated throughout the file?

Reply:
The documentation is about to change, after the OAI resource creation is incorporated into the playbook. Having said that, my comment is not that important.